ddl/syncer.go

```diff
@@ -38,11 +38,13 @@ const (
 	DDLGlobalSchemaVersion = "/tidb/ddl/global_schema_version"
 	// InitialVersion is the initial schema version for every server.
 	// It's exported for testing.
-	InitialVersion       = "0"
+	InitialVersion = "0"
+	// KeyOpDefaultRetryCnt is the default retry count for etcd store.
+	KeyOpDefaultRetryCnt = 3
```

Comment:
Why export these variables.

Reply:
I want to reuse these variables in domain/info.go.